parser/parser.y

```diff
@@ -430,6 +430,8 @@ import (
 	paramMarker	"?"
 	rsh		">>"
 
+%token<ident> or2
```

Comment:
what does `or2` means ?

Reply:
represent `||` when `PIPES_AS_CONCQT` is not set. I think it's better to distinguish `||`  under different sql_modes, and that's how MySQL do

Reply:
@winkyao PTAL